brainglobe_heatmap/heatmaps.py

```diff
@@ -250,6 +259,9 @@ def plot(
 
         if show_legend:
             ax.legend()
-        plt.show()
 
-        return f
+        if ax is None:
```

Comment:
I think to reproduce previous behaviour, you need to remember whether `ax is None` further up, because at this point you will have set it on L193?